test/spec/Feature/Query/InsertSpec.hs

```diff
@@ -489,13 +489,21 @@ spec actualPgVersion = do
         , matchHeaders = [matchContentTypeJson]
         }
 
-    context "with unicode values" $
+    context "with unicode values" $ do
+      it "succeeds and returns full representation" $
+        request methodPost "/simple_pk2?select=extra,k"
+            [("Prefer", "return=representation")]
+            [json| { "k":"圍棋", "extra":"￥" } |]
+        `shouldRespondWith`
+          [json|[ { "k":"圍棋", "extra":"￥" } ]|]
+          { matchStatus = 201 }
```

Comment:
Had to separate this test into two: Test returning unicode characters in the body with `return=representation` and in the Location header with `return=headers-only`.

Also, I noticed this is under the `describe "CSV insert"` group of tests, shouldn't the payload be in CSV format?

Reply:
Yeah, the payload can be changed or the test should be moved.

Reply:
There's a similar test for Updates under "Patching record" that was added at the same time: 

https://github.com/PostgREST/postgrest/blob/f7745e1569f47dc02cfd0e1e526b8be42293db6c/test/spec/Feature/Query/UpdateSpec.hs#L283-L289

So I moved this one under the "Posting new record" group for consistency.